Src/PCompiler/CompilerCore/Backend/Symbolic/SymbolicCodeGenerator.cs

```diff
@@ -80,12 +100,17 @@ private CompiledFile GenerateSource(CompilationContext context, Scope globalScop
             WriteMainDriver(context, source.Stream, globalScope, decls);
 
             context.WriteLine(source.Stream, "PTestDriver testDriver = null;");
+            context.WriteLine(source.Stream, "@Generated");
             context.WriteLine(source.Stream, "public PTestDriver getTestDriver() { return testDriver; }");
+            context.WriteLine(source.Stream, "@Generated");
             context.WriteLine(source.Stream, "public void setTestDriver(PTestDriver input) { testDriver = input; }");
             context.WriteLine(source.Stream);
 
+            context.WriteLine(source.Stream, "@Generated");
```

Comment:
What are these annotations?

Reply:
Are they used for some code coverage?

Reply:
Yes.
All of these functions marked as `@Generated` are helper functions appearing in Symbolic IR and does not include any model-specific business logic.
`@Generated` is a lombok annotation. When running PSym with code coverage provided through JaCoCo, functions annotated with `@Generated` are skipped in the coverage estimation and reporting by JaCoCo.